README.md

```diff
@@ -3,49 +3,96 @@
 
 [![Mentioned in Awesome Wind](https://awesome.re/mentioned-badge.svg)](https://github.com/IEA-Task-43/awesome-wind)
 
-# digital_wra_data_standard
+# Digital WRA Data Standards
 
-This repository is for the work on standardizing digital tools in wind resource assessment(WRA). This standard data model and associated tools are intended as universal building blocks for Wind Energy Resource Assessment applications.  Our goals for the project are to make the WRA process more efficient, more repeatable and more scalable.
+This repository is for the work, carried out by IEA Wind Task 43, on standardizing digital tools in wind resource 
+assessment (WRA). It currently consists of data models for wind resource measurements (_WRA Data Model_) and 
+digital calibration certificates. Along with associated tools, these are intended as universal building blocks for 
+wind energy yield assessment applications.
 
-## WRA Data Model
+## Mission
+_**"Our mission is to make the energy yield assessment process more efficient, transparent and reproducible 
+through digitizing and automation."**_
 
-A working group focuses on standardizing wind resource measurement data headers. The following tools are part of this undertaking:
+## What is a "Data Model"?
+A data model is an abstract model of real-world entities that organizes elements of data and standardizes how they 
+relate to one another. In this instance, the _WRA Data Model_ standardizes how properties of a met mast (e.g. 
```

Comment:
suggest making this language more broad than just met tower.  maybe resource measurement station or something generic like that.  since it includes RSD and solar.